File: spring-data-requery/src/main/java/org/springframework/data/requery/NotSupportedException.java
package org.springframework.data.requery;

/**
 * org.springframework.data.requery.NotSupportedException
 *
 * @author debop
 * @since 18. 6. 22
 */
public class NotSupportedException extends RuntimeException {

    public NotSupportedException() {}

    public NotSupportedException(String message) {
        super(message);
    }

    public NotSupportedException(String message, Throwable cause) {
        super(message, cause);
    }

    public NotSupportedException(Throwable cause) {
        super(cause);
    }

    private static final long serialVersionUID = -4471049196596338256L;
}

File: spring-data-requery/src/main/kotlin/org/springframework/data/requery/NotSupportedException.kt
package org.springframework.data.requery

/**
 * NotSupportedException
 *
 * @author debop@coupang.com
 * @since 18. 7. 2
 */
class NotSupportedException: RuntimeException {

    constructor(): super()
    constructor(message: String): super(message)
    constructor(message: String, cause: Throwable): super(message, cause)
    constructor(cause: Throwable): super(cause)
}